CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] Querier: it's no more required to set `-frontend.query-stats-enabled=true` in the querier to enable query statistics logging in the query-frontend. The flag is now required to be configured only in the query-frontend and the information will be propagated to the queriers in the internal request. #3595
```

Comment:
```suggestion
* [CHANGE] Querier: it's not required to set `-frontend.query-stats-enabled=true` in the querier anymore to enable query statistics logging in the query-frontend. The flag is now required to be configured only in the query-frontend and it will be propagated to the queriers. #3595
```